contrib/gtwvg/paint.prg

```diff
@@ -603,6 +603,10 @@ FUNCTION wvt_GetRGBColorByString( cColor, nForeBack )
             s := SubStr( cColor, 1, n - 1 )
          ELSE
             s := SubStr( cColor, n + 1 )
+            s := SubStr( cColor, n + 1 )
+            IF "," $ s
+               s := Substr( s, 1, At( ",", s ) - 1 )
```

Comment:
I'd use `Left` here, but if you prefer `SubStr`, then please write it in camel-case.